fabfile.py

```diff
@@ -63,7 +38,7 @@ def install_python_requirements():
 
 
 def chown_everything():
-    run(f"chown -R www-data:www-data {env.path}")
+    run(f"chown -R {env.user}:www-data {env.path}")
```

Comment:
Is this chowning necessary now?  It was before, because files were being created by the root user.

Reply:
Yes, something in a previous step was chowning as `ebmbot:ebmbot` and I figured I'd change as few things as possible!